torchft/process_group_test.py

```diff
@@ -553,6 +553,34 @@ def test_nccl_apis(self) -> None:
 
         torch.cuda.synchronize()
 
+    # pyre-fixme[56]: Pyre was not able to infer the type of argument
+    @skipUnless(
+        torch.cuda.is_available() and torch.cuda.nccl.version() >= (2, 25),
+        "needs NCCL >=2.25",
```

Comment:
Do we have any version check in the torchft source code to verify the NCCL version?

Reply:
@fegin we do have some checks (that gate usage of ncclCommAbort) but we don't have any hard checks. I was debating adding a hard dep on PT 2.7+ since 2.6 has many known issues.

Users should only use PT 2.7 RC or nightly